editor/jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     {
       testEnvironment: 'jest-environment-jsdom-global',
       preset: 'ts-jest',
-      testPathIgnorePatterns: ['/lib/', '/node_modules/'],
+      testPathIgnorePatterns: ['/lib/', '/node_modules/', '/.github-test-projects/'],
```

Comment:
Is this still needed / did this even make a difference?

Reply:
I guess there's no harm in keeping it.

Reply:
Potential confusion if it does nothing